Cargo.toml

```diff
@@ -10,6 +10,7 @@ authors = ["Ryan Leckey <leckey.ryan@gmail.com>"]
 categories = ["config"]
 license = "MIT/Apache-2.0"
 edition = "2018"
+rust-version = "1.56.0"
```

Comment:
As noted by https://github.com/mehcode/config-rs/pull/483#issuecomment-1775534431
- This should only reflect what the project can build successfully with (_even if that requires [managing a `Cargo.lock` with some pinning](https://github.com/mehcode/config-rs/pull/495#issuecomment-1793634940)_).
- It should only be raised in future for explicit changes to the project (_not from implicit changes from deps semver raising MSRV_):
  - Adding/Upgrading deps in `Cargo.toml` where pinning cannot maintain support for `rust-version`.
  - Introducing changes that require features from newer toolchains.

When it comes to actual support / maintenance the MSRV policy of the project may differ. Raising `rust-version` is a more granular `edition` field, but with support to opt-out / ignore it by downstream.